tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -920,4 +920,119 @@ public function provideInvalidPhpdocCases()
             ],
         ];
     }
+
+    public function testDescriptionAlignTag()
+    {
+        $this->fixer->configure(['description_align' => 'tag']);
+
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param int $a Desc
+     * ription
+     */
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * @param int $a Desc
+     *    ription
+     */
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testDescriptionAlignHint()
+    {
+        $this->fixer->configure(['description_align' => 'hint']);
+
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param int $a Desc
```

Comment:
Please add cases with e.g. `@method`, `@return`, ... and some cases with mixed tags. Same for other tests below.